api/src/main/java/marquez/db/models/ColumnLineageNodeData.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018-2022 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.db.models;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Getter
+@AllArgsConstructor
+public class ColumnLineageNodeData implements NodeData {
```

Comment:
_Minor_: Since we control the API response, we may consider using `dataset` instead of `name`? To me, that's a bit more clear, but it's also how OL defines the properties. I'd also use `fieldType` instead of `dataType` to better represent the pairing between the fields:`field`, `fieldType`